modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerTest.java

```diff
@@ -389,10 +353,15 @@ public void testPreconfiguredTable() throws Exception {
     /**
      * Tests create a table through public API.
      *
-     * @throws Exception If failed.
      */
     @Test
-    public void testCreateTable() throws Exception {
+    public void testCreateTable() throws NodeStoppingException {
```

Comment:
For a test, it doesn't seem to make sense to specify exact exceptions it throws. If it throws checked exception, Java compiler forces us to declare something, so let's just declare `throws Exception` in such cases. Details will just distract the reader